scripts/nginx/moviematch.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cat > /etc/nginx/apps/moviematch.conf << NGINX
+    location ^~ /moviematch/ {
+        proxy_pass http://127.0.0.1:8420/;
+        proxy_set_header Upgrade \$http_upgrade;
```

Comment:
You can set the root path in the nginx config like:

`proxy_set_header x-forwarded-prefix /moviematch;`